ext/http/01_http.js

```diff
@@ -521,7 +524,7 @@ function buildCaseInsensitiveCommaValueFinder(checkText) {
 
   /** @param value {string} */
   function skipWhitespace(value) {
-    while (char === spaceCharCode) {
+    while (ArrayPrototypeIncludes(spaceCharCodes, char)) {
```

Comment:
I would've gone with an explicit `char === spaceCharCode || char === tabCharCode` as it's _probably_ faster than `Array.prototype.includes` but the difference is likely very small if there is any at all.

Reply:
I'll update it